src/js/rendering/FramesheetRenderer.js

```diff
@@ -13,11 +13,13 @@
     }
   };
 
-  ns.FramesheetRenderer.prototype.renderAsCanvas = function () {
-    var canvas = this.createCanvas_();
+  ns.FramesheetRenderer.prototype.renderAsCanvas = function (framesPerRows) {
```

Comment:
"should be defaulted to 0 if it's not provided" well actually it should default to this.frames.length (or Infinity)